pysrc/SBBox.cpp

```diff
@@ -35,36 +35,40 @@ namespace galsim {
     struct PySBBox 
     {
 
-        static SBBox* construct(const bp::object & width, const bp::object & height, double flux,
```

Comment:
Oh I see, below is another more compact way to wrap the constructor.  Is there anything different about that functionally?